app/templates/about.jade

```diff
@@ -1,188 +1,363 @@
+          #story-graphic-1.col-lg-6
+            .media
+              .pull-left
+                img.media-object#story-image-1(src="/images/pages/about/Character_Silouhette.png")
+              .media-body
+                .media-heading
+                  | 5,000,000+
+                  br
+                  | total players
+                p have started their programming journey through CodeCombat
+          #story-graphic-2.col-lg-6
+            .media
+              p We’ve been translated into over 50 languages - our players hail from
+              .pull-right
+                img(src="/images/pages/about/Globe_Green.png")
+              .media-body
+                .media-heading 200+ countries
+                
+        #story-graphic-3
+          p
+            div.small-text Together, they have written
+            #story-bracketed-text
+              div
+                img(src="/images/pages/about/Bracket_left.png")
+              div.big-text
+                | 1 billion lines of code
+                br
+                | and counting
+              div
+                img(src="/images/pages/about/Bracket_right.png")
+          p
+            .small-text across six different programming languages
+            img.hidden-xs(src="/images/pages/about/Languages.png")
+            img.hidden-sm.hidden-md.hidden-lg(src="/images/pages/about/Languages_group1.png")
+            img.hidden-sm.hidden-md.hidden-lg(src="/images/pages/about/Languages_group2.png")
+        
+        #story-graphic-4
+          p
+            div.small-text Though we've come a long way...
+            figure
+              img(src="/images/pages/about/sketch.png")
+              figcaption
+                small Nick's very first sketch depicting a programming game in action.
+          p
+            .small-text we still have much to do before we complete our quest, so...
+        
+      #jobs.anchor
+        h3 Come work with us and help write CodeCombat history!
+        h4 Don't see a good fit but interested in keeping in touch? See our "Create Your Own" listing.
+        #jobs-row.row
+          .col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0
+            #benefits.center-block
+              h5 Employee Benefits
+              ul
+                li
+                  small Competitive salary and options
+                li
+                  small 15 day minimum vacation policy, excluding company holidays
+                li
+                  small Work from home flexibility
+                li
+                  small Unlimited sick/personal days
+                li
+                  small Professional development and continuing education support
+                li
+                  small Medical/dental/vision insurance
+          .col-sm-6.col-md-5.col-lg-4
+            .job-listing.center-block
+              h5 Software Engineer
+              small.label
+                | San Francisco • Fulltime
+              .row
```

Comment:
I think this button could be centered more simply, like wrapped in a .text-center.